### DISCUSSION TITLE
to delete

### DISCUSSION THREAD

**Discussion by Olyno** [community/community#13435] [Other Feature Feedback, Questions, & Ideas]
Immature content, to delete thanks you

**Comment by KlausEverWalkingDev** [comment]


**Reply by Olyno** [reply]
In fact, I wanted to make a fuss, as this is an immature discussion that has no place here. 
I wanted to moralize that this was not the right place, but I act the same way, hence the fact that it is better to delete this conversation that has no head or tail, and debate in the right places.

Anyway, I'm glad that some people agree with my way of thinking 😁
